src/base-testplane.ts

```diff
@@ -72,6 +69,6 @@ export abstract class BaseHermione extends AsyncEmitter {
     }
 
     protected _loadPlugins(): void {
-        pluginsLoader.load(this, this.config.plugins, PREFIX);
+        pluginsLoader.load(this, this.config.plugins, "hermione-");
```

Comment:
Using hard-coded `"hermione-"` here, because it no longer depends on `package.json`-s name, and we are not planning to add ability to use implicit plugin names in Testplane.